### PR TITLE
ITR beta: Mention it also works with the Agent

### PR DESCRIPTION
**Pull request by albertvaka**
### What does this PR do?
Remove note about requiring Agentless for ITR to work.

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Review by edrevo — CHANGES_REQUESTED** [review]
I think we need to remove the references to `DD_CIVISIBILITY_AGENTLESS_ENABLED=true` in the rest of the page, since it is not compulsory anymore.

Also, maybe we need to add a mention in the Swift section saying that agent support is not available?

**Comment by albertvaka** [comment]
That assumes we want to recommend "agentful" over agentless. Is that the case? I think there's less friction for customers to onboard with Agentless (since they don't need to install/upgrade their Agent), so I think the current instructions are okay. But it's not a strong opinion, I can change it if we want to favor the agentful setup.

**Comment by edrevo** [comment]
image

My interpretation of that is that an API_KEY is required, and that the DD_CIVISIBILITY_AGENTLESS_ENABLED must be set to true. Maybe we can change it to this?

```
DD_CIVISIBILITY_AGENTLESS_ENABLED
Enables or disables Agentless mode.
Default: false
```

**Review by drichards-87 — APPROVED** [review]
Approved from Docs. Let me know when the PR is ready to be merged.

**Comment by albertvaka** [comment]
I've made the changes you suggested @edrevo

File: content/en/continuous_integration/intelligent_test_runner/_index.md [hunk 1]
```diff
@@ -101,12 +100,11 @@ Intelligent test runner for Javascript skips entire _test suites_ (test files) r
 
 To enable Intelligent Test Runner, the version of the `dd-trace` tool must be >= 2.22.0 (execute `dd-trace --version` to get the version of the tool) and the following environment variables must be set:
 
-`DD_CIVISIBILITY_AGENTLESS_ENABLED=true` (Required)
+`DD_CIVISIBILITY_AGENTLESS_ENABLED=true`
```
Comment (edrevo) [inline content/en/continuous_integration/intelligent_test_runner/_index.md:103]:
```suggestion
`DD_CIVISIBILITY_AGENTLESS_ENABLED`
```

File: content/en/continuous_integration/intelligent_test_runner/_index.md [hunk 2]
```diff
@@ -45,12 +45,11 @@ Prior to setting up Intelligent Test Runner, you must have finished setting up [
 
 To enable Intelligent Test Runner, the following environment variables need to be set:
 
-`DD_CIVISIBILITY_AGENTLESS_ENABLED=true` (Required)
+`DD_CIVISIBILITY_AGENTLESS_ENABLED=true`
```
Comment (edrevo) [inline content/en/continuous_integration/intelligent_test_runner/_index.md:48]:
```suggestion
`DD_CIVISIBILITY_AGENTLESS_ENABLED`
```

**Review by edrevo — APPROVED** [review]

**Comment by albertvaka** [comment]
@drichards-87 this is ready to be merged 😃 Thanks!

**Pull request merged**